libtransmission/variant.cc

```diff
@@ -926,7 +926,7 @@ void tr_variantMergeDicts(tr_variant* target, tr_variant const* source)
         {
             tr_variant* t = nullptr;
 
-            // if types differ, ensure that target will overwrite source
+            // if types differ, ensure that source will overwrite target
```

Comment:
@ckerr I'm not 100% positive this is your original intention.

Just pointing it out for your attention.